tests/contrib/psycopg/test_psycopg_async.py

```diff
@@ -319,6 +319,18 @@ async def test_connection_execute(self):
         assert len(rows) == 1, rows
         assert rows[0][0] == "one"
 
+    @mark_asyncio
+    async def test_connection_context_execute(self):
```

Comment:
Can we also add test coverage for ``TracedAsyncCursor.__aexit__()``

Reply:
added.